types/shippo/index.d.ts

```diff
@@ -1,65 +1,501 @@
+        | 'LU'
+        | 'MO'
+        | 'MK'
+        | 'MG'
+        | 'MW'
+        | 'MY'
+        | 'MV'
+        | 'ML'
+        | 'MT'
+        | 'MH'
+        | 'MQ'
+        | 'MR'
+        | 'MU'
+        | 'YT'
+        | 'MX'
+        | 'FM'
+        | 'MD'
+        | 'MC'
+        | 'MN'
+        | 'ME'
+        | 'MS'
+        | 'MA'
+        | 'MZ'
+        | 'MM'
+        | 'NA'
+        | 'NR'
+        | 'NP'
+        | 'NL'
+        | 'NC'
+        | 'NZ'
+        | 'NI'
+        | 'NE'
+        | 'NG'
+        | 'NU'
+        | 'NF'
+        | 'MP'
+        | 'NO'
+        | 'OM'
+        | 'PK'
+        | 'PW'
+        | 'PS'
+        | 'PA'
+        | 'PG'
+        | 'PY'
+        | 'PE'
+        | 'PH'
+        | 'PN'
+        | 'PL'
+        | 'PT'
+        | 'PR'
+        | 'QA'
+        | 'RE'
+        | 'RO'
+        | 'RU'
+        | 'RW'
+        | 'BL'
+        | 'SH'
+        | 'KN'
+        | 'LC'
+        | 'MF'
+        | 'PM'
+        | 'VC'
+        | 'WS'
+        | 'SM'
+        | 'ST'
+        | 'SA'
+        | 'SN'
+        | 'RS'
+        | 'SC'
+        | 'SL'
+        | 'SG'
+        | 'SX'
+        | 'SK'
+        | 'SI'
+        | 'SB'
+        | 'SO'
+        | 'ZA'
+        | 'GS'
+        | 'SS'
+        | 'ES'
+        | 'LK'
+        | 'SD'
+        | 'SR'
+        | 'SJ'
+        | 'SZ'
+        | 'SE'
+        | 'CH'
+        | 'SY'
+        | 'TW'
+        | 'TJ'
+        | 'TZ'
+        | 'TH'
+        | 'TL'
+        | 'TG'
+        | 'TK'
+        | 'TO'
+        | 'TT'
+        | 'TN'
+        | 'TR'
+        | 'TM'
+        | 'TC'
+        | 'TV'
+        | 'UG'
+        | 'UA'
+        | 'AE'
+        | 'GB'
+        | 'UM'
+        | 'UY'
+        | 'UZ'
+        | 'VU'
+        | 'VE'
+        | 'VN'
+        | 'VG'
+        | 'VI'
+        | 'WF'
+        | 'EH'
+        | 'YE'
+        | 'ZM'
+        | 'ZW';
+
+    type DistanceUnit = 'cm' | 'in' | 'ft' | 'mm' | 'm' | 'yd';
+    type MassUnit = 'g' | 'oz' | 'lb' | 'kg';
+
+    interface ValidationError {
+        source: string;
+        code: string;
+        type: string;
+        text: string;
+    }
+
+    interface Metadata {
+        object_created: string;
+        object_updated: string;
+        object_id: string;
+        object_owner: string;
+    }
+
+    interface Message {
+        code?: string;
+        message: string;
+    }
+
     // https://goshippo.com/docs/reference#addresses
-    interface Address {
-        city?: string | undefined;
+    interface Address extends Metadata {
+        name: string;
         company?: string | undefined;
-        country?: string | undefined;
-        name?: string | undefined;
         phone?: string | undefined;
-        state?: string | undefined;
+        email?: string | undefined;
         street1: string;
+        street_no?: string | undefined;
         street2?: string | undefined;
         street3?: string | undefined;
+        city: string;
+        state: string;
```

Comment:
These changed to required (and `name`); was that intentional?

Reply:
Yes - see api docs [here](https://goshippo.com/docs/reference#addresses-create). Since these fields are required to create an address in shippo, they can be guaranteed to be present in every fetch.